source/documentation/virtual_machine_management_guide/topics/Using_a_snapshot_to_restore_a_virtual_machine.adoc

```diff
@@ -9,13 +9,13 @@ A snapshot can be used to restore a virtual machine to its previous state.
 
 . Click menu:Compute[Virtual Machines] and select a virtual machine.
 . Click the virtual machine's name to go to the details view.
```

Comment:
Suggest: "The name of the virtual machine" as IBM frowns on using the possessive with inanimate obje4cts.